react-function-components/src/index.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom';
+
+function CustomButton() {
+  return (
+    <button>
+      Click Me
+    </button>
+  );
+}
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render(CustomButton());
```

Comment:
Here we don't want to call our component (in react we pretty much never call our own components. Instead, we use them to create a react element

```jsx
const element = <CustomButton />;
```